### ISSUE TITLE
Add DDN-informed workload example presets for BOM input auto-population

### ISSUE THREAD

**Issue by jwsiejk** [jwsiejk/accountlist#352]
## Goal
Add workload-specific example presets to the AI Workload Discovery Mapper so the user can auto-populate the **BOM Input Capture** and relevant questionnaire fields with realistic illustrative inputs for the currently selected workload.

## Important framing
DDN has public use cases and reference architecture content. Use those as directional guidance for the kinds of inputs and architecture assumptions that make sense, but do **not** claim exact DDN sizing, exact DDN SKUs, exact GPU counts, or official DDN BOM guidance unless the repo already contains explicit approved source content.

The examples must be presented as:

> Illustrative discovery inputs based on common AI/HPC/FSI workload patterns and DDN-style reference architecture thinking — not final sizing recommendations or final BOM guidance.

## Why
The tool currently identifies BOM readiness gaps. We now want a strong demo flow:

1. Select a workload.
2. Click an auto-populate example button.
3. BOM Input Capture fills with realistic discovery values.
4. BOM readiness increases.
5. Known inputs and talk track become more concrete.

This helps show how the tool can move from blank discovery to a realistic workload mapping conversation.

## Scope
Focus on the AI Workload Discovery Mapper.

Likely files:

```text
ui/partner-hub/components/workload-mapper/workload-mapper.tsx
ui/partner-hub/components/workload-mapper/workload-mapper-data.ts
ui/partner-hub/components/workload-mapper/workload-mapper-types.ts
ui/partner-hub/components/workload-mapper/workload-mapper-utils.ts
```

Follow existing TypeScript/React/Next.js patterns and preserve the readability improvements from recent commits.

### 1. Add example presets per workload
For every preloaded workload, define at least one realistic example preset.

Preloaded workloads:

FSI:
- Fraud Detection & Investigation
- Risk Modeling / Stress Testing
- Trading / Quant Research
- Compliance / Document Intelligence
- RAG Knowledge Assistant
- Model Training / Fine-tuning

HPC / AI:
- Large-scale AI Training
- Inference Platform
- Scientific Simulation
- Genomics / Life Sciences
- Media / Rendering Pipeline

Custom:
- Provide a generic custom AI workload example, or cleanly disable presets for custom workloads with explanatory text.
- Preferred: provide a generic custom example.

### 2. DDN-informed but not DDN-official
Use DDN use cases/reference architecture thinking directionally:

- AI training workloads should emphasize GPU utilization, high-throughput training storage, checkpointing, distributed training, and benchmark/tuning inputs.
- RAG/document intelligence workloads should emphasize ingestion, chunking/embedding, vector retrieval, governance, data freshness, and access control.
- FSI workloads should emphasize regulated data, auditability, retention, intraday/near-real-time needs, and explainability.
- HPC simulation workloads should emphasize scheduler, parallel file/data platform, high-speed interconnect, simulation output, and post-processing.
- Inference workloads should emphasize concurrency, latency SLOs, model serving, cache/session layer, guardrails, and observability.

Do not include exact product sizing like “buy X AI400X nodes” or “Y H100s” unless verified and explicitly framed as an example only. Prefer phrases like:

- `GPU pool required; exact count depends on model size, batch size, and training window`
- `High-performance data platform required; final design depends on throughput and metadata profile`
- `High-speed fabric required for distributed training; architecture validated through benchmark`

### 3. Presets should populate BOM sizing fields
Populate:

- exact data volume
- daily ingest rate
- file/object count
- query concurrency
- model size
- GPU requirement
- retention period
- HA/DR requirements
- preferred vendors
- budget/timeline

### 4. Presets should also populate relevant questionnaire fields
Where appropriate, populate:

- process / decision being improved
- success criteria
- AI pattern
- data types
- data size range
- daily ingest range
- file/object pattern
- freshness requirement
- performance tier
- latency requirement
- GPU dependency
- data sensitivity
- audit trail
- encryption
- data residency
- retention
- explainability
- access controls

### 5. Add UX near BOM Input Capture
Add a control in or near the **BOM Input Capture** card.

Suggested UX:

```text
Example preset
[Auto-populate realistic example for this workload]
```

Add visible disclaimer:

```text
Examples are illustrative discovery inputs informed by common workload/reference architecture patterns. They are not DDN official sizing, final BOM guidance, or SKU recommendations.
```

### 6. Preserve user control
After applying an example:

- all fields remain editable
- user can overwrite any field
- selected workload remains unchanged
- do not lock or disable populated values
- do not overwrite custom workload name unless applying the generic custom example intentionally does so

### 7. Example data guidance
Use realistic but clearly illustrative values.

#### Example: FSI Model Training / Fine-tuning
Use values like:

- exact data volume: `120 TB curated training corpus + 25 TB validation/test data`
- daily ingest rate: `2–5 TB/day new or updated domain data`
- file/object count: `~40 million documents/objects after chunking and curation`
- query concurrency: `50 internal researchers / batch evaluation jobs`
- model size: `7B–70B parameter open-weight model; LoRA/QLoRA fine-tuning initially`
- GPU requirement: `GPU pool required; exact count depends on model size, batch size, and training window`
- retention period: `7 years for governed datasets and experiment artifacts`
- HA/DR requirements: `Production registry and governed datasets replicated; training scratch tier recoverable from source`
- preferred vendors: `NVIDIA GPU stack, high-performance data platform, vector/model registry tooling`
- budget/timeline: `Pilot in 8–12 weeks; phased production expansion after benchmark`

#### Other examples should be similarly realistic
For example:

Fraud Detection & Investigation:
- mixed structured transaction data + unstructured case notes
- near-real-time/intraday freshness
- high sensitivity, strict audit trail, role-based access
- illustrative ingest and concurrency values

Risk Modeling / Stress Testing:
- historical market/risk datasets
- batch and intraday scenario runs
- large sequential/parallel reads
- governed retention and reproducibility

Trading / Quant Research:
- time-series heavy datasets
- high concurrency from researchers/jobs
- low-latency data access
- fast iteration/backtesting

Compliance / Document Intelligence:
- millions of regulatory/policy/legal docs
- chunking/embedding/vector retrieval
- strict auditability, data residency, explainability

RAG Knowledge Assistant:
- many small documents
- frequent incremental updates
- vector retrieval freshness
- RBAC/ABAC controls

Large-scale AI Training:
- PB-scale corpus possible
- distributed preprocessing
- checkpointing
- high-throughput training storage
- high GPU dependency

Inference Platform:
- latency SLOs
- request concurrency
- model cache/session state
- guardrails and observability

Scientific Simulation:
- parallel file/data platform
- scheduler
- high-speed interconnect
- simulation outputs and visualization

Genomics / Life Sciences:
- many small files
- sequencing pipeline throughput
- long retention/reproducibility
- regulated data controls

Media / Rendering Pipeline:
- mixed file sizes
- throughput orchestration
- deadline-driven scale
- high ingest during production windows

### 8. Output impact
After applying a preset:

- BOM readiness should increase because fields are populated
- known inputs should display the example values
- missing inputs should reduce accordingly
- talk track should reflect populated details where current logic supports it

### 9. Code quality
- Keep TypeScript strongly typed
- No `any`
- Keep preset data in a clean data structure, preferably in `workload-mapper-data.ts`
- Add/adjust types in `workload-mapper-types.ts` if needed
- Preserve readable, multi-line JSX from recent cleanup
- No external API calls
- No fake backend
- No stale code
- Do not reintroduce CHIP-8 or Steam Trains references

## Validation
Run from the correct package directory:

```bash
npm run lint
npm run typecheck
npm test -- --runInBand
npm run build
rg -i "chip-?8|steam[- ]?trains" . || true
```

If any command does not exist, document that clearly and run the closest available equivalent.

## Acceptance criteria
- Each preloaded workload has an example preset
- BOM Input Capture can auto-populate realistic values for the selected workload
- Presets are DDN-informed directionally but not represented as official DDN sizing/BOM guidance
- Fields remain editable after auto-population
- BOM readiness updates after applying a preset
- Example disclaimer is visible
- Custom workload behavior is handled cleanly
- Typecheck/lint/build pass or unavailable commands are documented
- No stale CHIP-8 / Steam Trains references are introduced

## Commit / PR summary should include
- example preset data added
- DDN-informed/non-official framing
- auto-populate UX added
- custom workload handling
- validation output
- stale-reference scan result
